EXILED/Exiled.API/Features/Spawn/RoomSpawnPoint.cs

```diff
@@ -48,6 +48,11 @@ public override Vector3 Position
             {
                 Room roomInstance = Features.Room.Get(Room) ?? throw new InvalidOperationException("The room instance could not be found.");
 
+                if (roomInstance.Type == RoomType.Surface)
+                {
+                    return Offset != Vector3.zero ? Offset : roomInstance.Position;
+                }
```

Comment:
```suggestion
                if (roomInstance.Type == RoomType.Surface)
                    return Offset != Vector3.zero ? Offset : roomInstance.Position;
```

Since it's 1 line, no need for the brackets.